src/actions/auth_actions/confirm_login_action.js

```diff
@@ -12,8 +14,9 @@ Actions.register(CONFIRM_LOG_IN, payload => {
   })
   .then(checkStatus)
   .then(response => {
-    authStore.setAuthed(response.data.token);
-    authStore.setIsUserNew(response.data.is_new);
+    googleAuthStore.setAuthed(response.data.token);
+    googleAuthStore.setIsUserNew(response.data.is_new);
```

Comment:
Seems nice that we're specifying that this is Google Auth if we're considering adding in Facebook auth as well.

Reply:
Could you explain what the auth_mall is doing on line 6?

Reply:
ah, it's an unused import. I can remove it